backup/backup.go

```diff
@@ -79,19 +138,45 @@ func (b *BackupManager) convertEtcdToBackup(resp []*mvccpb.KeyValue) backup {
 	return values
 }
 
-func (f backup) writeTo(filename string) error {
-	file, err := json.Marshal(f)
+func (b *BackupManager) loadSnapshot(filename string) error {
+	backupData := backup{}
+	err := backupData.readFrom(filename)
 	if err != nil {
-		return fmt.Errorf("could not marshal snapshot: %v", err)
+		return fmt.Errorf("could not read from file %s: %v", filename, err)
 	}
-	if err = ioutil.WriteFile(filename, file, 0644); err != nil {
-		return fmt.Errorf("could not write snapshot to file: %v", err)
+	fmt.Println("Clearing ETCD")
```

Comment:
on purpose?

Reply:
Left it so the user has some reference to if its running rather than waiting for an empty terminal. Can change it to be a log too

Reply:
Can we make it a log please

Reply:
Changed to logs